lib/blocs/profile/profile_cubit.dart

```diff
@@ -179,7 +179,9 @@ class ProfileCubit extends Cubit<ProfileState> {
   Future<void> deleteTables() async {
     // Delete all table data.
     await _db.transaction(() async {
+      print('Is table empty: ${_db.allTables.isEmpty}');
       for (final table in _db.allTables) {
+        print('deleting table: ${table.actualTableName}');
```

Comment:
Do we want these logs here? If not then...
```suggestion
      for (final table in _db.allTables) {
```